stubs/torch/distributed/__init__.pyi

```diff
@@ -62,8 +62,8 @@ def all_reduce(tensor: Tensor, op: ReduceOp = ReduceOp.SUM, group:Optional[Proce
 def all_gather(tensor_list: List[Tensor], tensor: Tensor, group:Optional[ProcessGroup] = None, async_op: bool = False): ...
 def reduce_scatter(tensor: Tensor, input_list: List[Tensor], op: ReduceOp = ReduceOp.SUM, group:Optional[ProcessGroup] = None, async_op: bool = False): ...
 # These two functions takes flatten tensors directly, avoiding internal buffer allocations overheads.
-def _all_gather_base(input_tensor: Tensor, output_tensor: Tensor, group:Optional[ProcessGroup] = None): ...
-def _reduce_scatter_base(output_tensor: Tensor, input_tensor: Tensor, group:Optional[ProcessGroup] = None): ...
+def _all_gather_base(input_tensor: Tensor, output_tensor: Tensor, group:Optional[ProcessGroup] = None, async_op: bool = False): ...
```

Comment:
these 2 lines of updates are great. If you want, you can separate them into a PR and get reviewed and merge it in first.